data-prepper-plugins/elasticsearch/security.md

```diff
@@ -40,39 +40,33 @@ You should ensure that the credentials you configure have the required permissio
 
 Please check this [doc](https://docs.aws.amazon.com/elasticsearch-service/latest/developerguide/es-ac.html) to know how to set IAM to your Elasticsearch domain,
 
-### Limitations
-
-* The Elasticsearch sink will not work with [custom endpoint](https://docs.aws.amazon.com/elasticsearch-service/latest/developerguide/es-customendpoint.html) domains.
-* The Elasticsearch sink will not with domains that use both IAM and FGAC. (Should be available in next release. :) )
-
-## Opendistro For Elasticsearch
-
-Elasticsearch sink can send data to opendistro-for-elasticsearch (ODFE) cluster by administrative credentials as follows:
-
-```
-sink:
-  - elasticsearch:
-      ...
-      username: "admin"
-      password: "admin"
-```
+### Fine-Grained Access Control in Amazon Elasticsearch Service
+
+The Elasticsearch sink creates ISM policy for Trace Analytics indices and Amazon Elasticsearch service allows only the `master user` to create ISM policy. So,
```

Comment:
Suggest 
1. unabbreviating ISM on first use 
2. link off to the docs: https://opendistro.github.io/for-elasticsearch-docs/docs/ism/
2. Using ", but" instead of "and"
3. capitalizing Service

>_The Elasticsearch sink creates an [Index State Management (ISM)]() policy for Trace Analytics indices, but Amazon Elasticsearch Service allows only the `master user` to create the ISM policy._